hermes-guide-days-website/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,37 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
add this in index.html if you are going to use it to have it available globally in app